printer.go

```diff
@@ -9,13 +9,26 @@ import (
 	"io"
 	"math"
 	"os"
-	"strconv"
-	"strings"
 	"time"
 
 	"github.com/gonum/floats"
 )
 
+var printerHeadings = [...]string{
+	"Iter",
+	"Runtime",
+	"FuncEvals",
+	"GradEvals",
+	"HessEvals",
+	"Func",
+	"|Gradient|",
```

Comment:
Done.

I don't think non-ascii is a problem, printing microseconds in default format already uses μ anyway.